lib/tests/tests.c

```diff
@@ -346,7 +347,7 @@ test_single_locus_historical_sample(void)
     ret = tsk_table_collection_init(&tables, 0);
     CU_ASSERT_EQUAL_FATAL(ret, 0);
 
-    for (j = 0; j < 2; j++) {
+    for (j = 1; j < 2; j++) {
```

Comment:
I think these changes are brought in because you merged upstream rather than rebased. Git is lying to you when it tells you you should merge from an upstream remote. Always rebase...

Reply:
I've changed the for loop back to for (j = 0; j < 2; j++) {

Is that enough, or do I need to do something fancier to set the record straight?

Reply:
No, you still need to rebase to get the history sorted out. See the documentation [here](https://stdpopsim.readthedocs.io/en/latest/development.html#github-workflow) for guidance. We shouldn't see any merge commits in the history for this PR. The [network view](https://github.com/tskit-dev/msprime/network) is helpful for visualising. Your branch should fork off once, directly from the tip of upstream/master.